app/controllers/members_controller.rb

```diff
@@ -0,0 +1,45 @@
+class MembersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_project_by_id, only: [:index, :new, :create,:destroy]
+  before_action :find_member, only: [:destroy]
+
+  def index
+    @members = @project.members
+  end
+
+  def new
+    @members = User.where.not(id: @project.members.pluck(:user_id).compact)
+  end
+
+  def create
+    members_params[:user_id].each do |id|
+      @members = @project.members.create(user_id: id)
+    end
+
+    redirect_to action: "index"
+  end
+
+  def edit
+    
+  end
+
+  def update
+  end
+
+  def destroy
+    @member.destroy if @project.members.count > 1
```

Comment:
I guess, we should show some message to the user, if destroy was successfull  or not.

Reply:
done